src/lib/ruby-generator/music.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * Define Ruby code generator for Music Blocks
+ * @param {RubyGenerator} Generator The RubyGenerator
+ * @return {RubyGenerator} same as param.
+ */
+export default function (Generator) {
+    Generator.music_playDrumForBeats = function (block) {
+        const drum = Generator.valueToCode(block, 'DRUM', Generator.ORDER_NONE) || null;
+        const beats = Generator.valueToCode(block, 'BEATS', Generator.ORDER_NONE) || 0;
+        return `play_drum(drum: ${drum}, beats: ${beats})\n`;
+    };
+
+    Generator.music_menu_DRUM = function (block) {
+        const drum = Generator.quote_(Generator.getFieldValue(block, 'DRUM') || '1');
+        return [drum, Generator.ORDER_ATOMIC];
+    };
+
+    Generator.music_restForBeats = function (block) {
+        const beats = Generator.valueToCode(block, 'BEATS', Generator.ORDER_NONE) || 0;
+        return `rest(${beats})\n`;
+    };
+
+    Generator.music_playNoteForBeats = function (block) {
+        const note = Generator.valueToCode(block, 'NOTE', Generator.ORDER_NONE) || 0;
+        const beats = Generator.valueToCode(block, 'BEATS', Generator.ORDER_NONE) || 0;
+        return `play_note(note: ${note}, beats: ${beats})\n`;
+    };
+
+    Generator.music_playNoteForBeats = function (block) {
+        const note = Generator.valueToCode(block, 'NOTE', Generator.ORDER_NONE) || 0;
+        const beats = Generator.valueToCode(block, 'BEATS', Generator.ORDER_NONE) || 0;
+        return `play_note(note: ${note}, beats: ${beats})\n`;
+    };
+
+    Generator.note = function (block) {
+        const note = Generator.getFieldValue(block, 'NOTE') || 0;
+        return [note, Generator.ORDER_ATOMIC];
+    };
+
+    Generator.music_setInstrument = function (block) {
+        const instrument = Generator.valueToCode(block, 'INSTRUMENT', Generator.ORDER_NONE) || null;
+        return `set_instrument(${instrument})\n`;
+    };
+
+    Generator.music_menu_INSTRUMENT = function (block) {
+        const instrument = Generator.quote_(Generator.getFieldValue(block, 'INSTRUMENT') || '1');
```

Comment:
instrumentは数値がいいと思います。

```suggestion
        const instrument = Generator.getFieldValue(block, 'INSTRUMENT') || 1;
```

Reply:
![screenshot from 2018-12-05 13-23-03](https://user-images.githubusercontent.com/23467008/49490008-1a4cfe80-f891-11e8-90a4-8f51c6255f49.png)
こういうブロックなので数値ではなく文字列にしました。
case文で"(1)ピアノ"と返す方が良かったりしますか？　数値でよければ数値で良いとは思います!!

Reply:
数値でいいです。
Ruby -> Blocksにするときに、数値で決めうちだと楽だからです。ユーザーがタイプするのも数値だと楽だしね。